src/Foundation/NSDictionary_2.cs

```diff
@@ -147,7 +147,28 @@ public TValue[] ObjectsForKeys (TKey[] keys, TValue marker)
 			return Runtime.GetNSObject<NSDictionary<TKey,TValue>> (_FromObjectsAndKeysInternal (objects.Handle, keys.Handle));
 		}
 
-		public static NSDictionary<TKey,TValue> FromObjectsAndKeys (TKey [] objects, TValue [] keys)
+		public static NSDictionary<TKey, TValue> FromObjectsAndKeys (TValue [] objects, TKey [] keys, nint count)
```

Comment:
The point is that there already is an overload for NSObject, so if this type is created with TValue and TKey as NSObject, then there will be two overloads with the same signature, and none of them can be used.